docs/release-notes/helm-requires-initial-password.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Breaking Changes**
+
+-  Helm: a ``initialUserPassword`` entry is now required; see :ref:`Helm Chart
+   <helm-config-reference>`
```

Comment:
:orphan:

**Breaking Changes**

-  Helm: An entry for ``initialUserPassword`` is now required. See :ref:`Helm Chart <helm-config-reference>`.